Desktop/components/JASP/Widgets/PlotEditor/PlotEditingAxis.qml

```diff
@@ -80,8 +82,7 @@ Column
 
 			onClicked:
 			{
-				if (axisBreaksRange.checked) axisModel.breaksType = AxisModel.BreaksRange
-				else if (axisBreaksManual.checked) axisModel.breaksType = AxisModel.BreaksManual
+				showAxis.lastBreakType = axisModel.breaksType = axisBreaksRange.checked ? AxisModel.BreaksRange : AxisModel.BreaksManual
```

Comment:
Is it supposed to be `axisModel.breaksType = axisBreaksRange.checked` or `axisModel.breaksType == axisBreaksRange.checked`

Or `showAxis.lastBreakType = axisModel.breaksType = (axisBreaksRange.checked ? AxisModel.BreaksRange : AxisModel.BreaksManual)`

The ambiguity is a bit confusing. I thinkadding parentheses would be a golden move

Reply:
I see... a second line to assign lastBreakType might be better.